### ISSUE TITLE
Add basic js functions

### ISSUE THREAD

**Issue by synw** [timonweb/django-bulma#10]
Bulma does not come with the required javascript for some things, unlike Bootstrap. It is necessary to write it to get modals, menu toggle, tabs to work ...

I've seen a Bulmajs lib on github but all the examples are in ES6 and I dislike the api. It should not be difficult to write a few simple functions to do the job.

Would you be open to integrate basic js functions in this module so that we get an all in one ready to use Bulma engine for Django? I can contribute: I need some of these now and I am going to start writing them.

**Comment by timonweb** [comment]
Hello!

While I like the idea, I'm inclined to keep this project as simple as possible. 

You see, Bulma is just a CSS framework. It can be used with jQuery, React, Vue, Angular, or any other JS framework. And every of frontend frameworks uses different approaches to solving common problems.

I don't want to support any js code within this package. But as I said, I like your idea, and it would be great if you could release your javascript code as an add-on to django-bulma and name it, for example, django-bulma-js or something similar.

That way we'd get the js addon for django-bulma, and I wouldn't have to curate the javascript part of the package.

What do you say?

**Comment by synw** [comment]
>  It can be used with jQuery, React, Vue, Angular, or any other JS framework

I was thinking about super basic vanilla ES5 functions. I don't want to maintain a lot of javascript either so I want to keep it simple and non opinionated

I'll go for the add-on way then

**Comment by timonweb** [comment]
Sure, when you're ready, post the repo link here and I'll add it to the Readme.

**Comment by synw** [comment]
Ok I started a [bulma-extra](https://github.com/synw/bulma-extra) repo.

It contains js and a bit of css. It is actually very minimalistic, handling just the navburger. Tabs support will come in soon. I only add things when needed in this repository, to avoid maintaining useless things and avoid unnecessary bloat.

The repo is not tight to Django as I need this outside of Django as well. I made a [management command](https://github.com/synw/django-chartflo/blob/master/chartflo/management/commands/bulma_extra.py) to download and install the files in an easy way: 

`python3 manage.py bulma_extra static/css static/js` 

The params are the destinations for css and js files.

The command is currently located in a module where I need it now, but it would be more portable if in a generic module. Would it be possible to integrate it in django-bulma?
